packages/sanity/src/core/store/_legacy/document/document-pair/serverOperations/publish.ts

```diff
@@ -20,7 +20,7 @@ export const publish: OperationImpl<[], DisabledReason> = {
       throw new Error('cannot execute "publish" when draft is missing')
     }
 
-    return actionsApiClient(client).observable.action(
+    return actionsApiClient(client, idPair).observable.action(
```

Comment:
I think we should make a note to remove these added params after backend APIs stabilize